frontends/web/src/routes/account/send/send.tsx

```diff
@@ -637,11 +625,11 @@ class Send extends Component<Props, State> {
                         disabled={!this.getValidTxInputData() || !valid || isUpdatingProposal}>
                         {t('send.button')}
                       </Button>
-                      <ButtonLink
-                        secondary
-                        to={`/account/${code}`}>
+                      <BackButton
+                        disabled={activeCoinControl || activeScanQR}
+                        enableEsc>
```

Comment:
Setting the button to disabled it not really needed as the UI elements are covered by either coin-control or qr-scan popups.

Similar situation as in receive, but there it currently only conditionally sets enableEsc `<BackButton enableEsc={!addressTypeDialog && !verifying}>`